source/plugins/go/src/oms.go

```diff
@@ -1927,7 +1929,7 @@ func InitializePlugin(pluginConfPath string, agentVersion string) {
 	MdsdInsightsMetricsTagRefreshTracker = time.Now()
 	MdsdContainerLogTagRefreshTracker = time.Now()
 	Log("ContainerLogsRouteADX: %v, IsWindows: %v, IsAADMSIAuthMode = %v IsGenevaLogsIntegrationEnabled = %v \n", ContainerLogsRouteADX, IsWindows, IsAADMSIAuthMode, IsGenevaLogsIntegrationEnabled)
-	if !ContainerLogsRouteADX && IsWindows && IsAADMSIAuthMode && !IsGenevaLogsIntegrationEnabled {
+	if !ContainerLogsRouteADX && IsWindows && IsAADMSIAuthMode {
```

Comment:
do we need to add check for skip cert onboarding if the IsGenevaLogsIntegrationEnabled && IsAADMSIAuthMode

Reply:
> do we need to add check for skip cert onboarding if the IsGenevaLogsIntegrationEnabled && IsAADMSIAuthMode
No,   The logic what we have now is. 

if (IsAADMSIAuthMode) { skip cert generation}
else {
    generate cert
}

This covers regardless geneva enabled or not.